Updating path for validator keys (#2664)
XRPLF/xrpl-dev-portal: docs/infrastructure/configuration/server-modes/run-rippled-as-a-validator.md

In a secure location **not** on your validator:

> 1. Generate a validator key pair using the `create_keys` command. You can find it in the rippled package:

Reviewer: - Keep `rippled` in monospace font
- Calling it the `create_keys` command makes it seem like that might be the name of the executable on the commandline, so I think it makes more sense to clarify the name of the main executable and let the example show that `create_keys` is the argument you pass to it.

```suggestion
1. Generate a validator key pair using the `validator-keys` tool, which is included in the `rippled` package:
```
